vault/auth.go

```diff
@@ -73,7 +73,7 @@ func (c *Core) enableCredential(entry *MountEntry) error {
 		return fmt.Errorf("token credential backend cannot be instantiated")
 	}
 
-	if match := c.router.MatchingMount(credentialRoutePrefix + entry.Path); match != "" {
+	if match := c.router.MountConflict(credentialRoutePrefix + entry.Path); match != "" {
```

Comment:
For ease of understanding, can you change `match` here to `conflict` and in the error message say "conflicting mount at ..."?